src/reducers/index.js

```diff
@@ -1,21 +1,18 @@
 import { INCREMENT, DECREMENT } from '../actions';
 
-const initialState = {
-  count: 0
-}
 
 // Our reducer that handles our two action cases:
 // increment and decrement. It receives the state
 // of our redux store, along with an action created
 // by our action creator. What does the reducer
 // need to do with the count in each case?
-export default (state = initialState, action) => {
+export default (count = 0, action) => {
```

Comment:
`count = 0` works for this project bec of its scale. However, it's still good practice to define an `initialState` object and use `state = initialState`